model/training.py

```diff
@@ -28,8 +28,15 @@ def parse_args():
     parser.add_argument("--dataset_file", dest="data_json", type=str)
     parser.add_argument("--model_output_directory", dest="model_dir", type=str)
     parser.add_argument("--num_epochs", dest="num_epochs", type=int)
+    parser.add_argument(
+        "--labels",
+        dest="labels",
+        type=str,
+        required=False,
+        help="Slash(/)-separated list of labels",
```

Comment:
Instead of a slash separate list, we can just nargs in argparse. Here's an example https://stackoverflow.com/a/32763023